phpunit.xml.dist

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit colors="true" bootstrap="vendor/autoload.php">
+    <php>
```

Comment:
I'd rather not add the credentials here to avoid someone commiting them by mistake. You can set them in your environment via `.bash_profile` or `.bashrc` or define them like 
```
ALGOLIA_APP_ID=MY_APP_ID ALGOLIA_API_KEY=MY_API_KEY ./vendor/bin/phpunit
```